### PR TITLE
Update tiny model info. and pipeline testing

### PR DESCRIPTION
**Pull request by ydshieh**
# What does this PR do?

Just a regular update.

### CODE REVIEW

File: tests/models/t5/test_modeling_t5.py [hunk 1]
```diff
@@ -583,6 +583,16 @@ def setUp(self):
...
 
+    # `QAPipelineTests` is not working well with slow tokenizers (for some models) and we don't want to touch the file
+    # `src/transformers/data/processors/squad.py` (where this test fails for this model)
+    def is_pipeline_test_to_skip(
+        self, pipeline_test_casse_name, config_class, model_architecture, tokenizer_name, processor_name
+    ):
+        if pipeline_test_casse_name == "QAPipelineTests" and not tokenizer_name.endswith("Fast"):
+            return True
+
+        return False
```
Comment (ydshieh) [inline tests/models/t5/test_modeling_t5.py:594]: Just for you to read this message @amyeroberts

**Review by amyeroberts — APPROVED** [review]
LGTM - thank for updating these!

File: tests/models/layoutlmv3/test_modeling_tf_layoutlmv3.py [hunk 2]
```diff
@@ -277,11 +277,7 @@ class TFLayoutLMv3ModelTest(TFModelTesterMixin, PipelineTesterMixin, unittest.Te
         if is_tf_available()
         else ()
     )
-    pipeline_model_mapping = (
-        {"document-question-answering": TFLayoutLMv3ForQuestionAnswering, "feature-extraction": TFLayoutLMv3Model}
-        if is_tf_available()
-        else {}
-    )
+    pipeline_model_mapping = {"feature-extraction": TFLayoutLMv3Model} if is_tf_available() else {}
```
[the remark this answers is not in the document]
Comment (ydshieh) [reply tests/models/layoutlmv3/test_modeling_tf_layoutlmv3.py:280]: Good catch!

I think I added it in #24364 manually (by looking the torch counterpart), but forgot to update the auto mapping names. So this time when I ran the script to update things in an automatic way, it got erased. 

To keep it in `"document-question-answering"`, `TFLayoutLMv3ForQuestionAnswering` needs to be in `TF_MODEL_FOR_DOCUMENT_QUESTION_ANSWERING_MAPPING_NAMES` (which is not currently).

I will revert the change here, but fix this mapping thing in  another PR.

[the remark this answers is not in the document]
Comment (ydshieh) [reply tests/models/layoutlmv3/test_modeling_tf_layoutlmv3.py:280]: Look [that comment](https://github.com/huggingface/transformers/pull/24364/files#r1234925865), it doesn't really matter (at this moment).

**Pull request merged**